db/schema.rb

```diff
@@ -232,7 +232,9 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "google_user_id"
+    t.string "lms_user_id"
     t.index ["google_user_id"], name: "index_roster_entries_on_google_user_id"
+    t.index ["lms_user_id"], name: "index_roster_entries_on_lms_user_id"
```

Comment:
Looks like there's DB changes in the PR! I'm assuming these aren't supposed to be here, please ping me when they're cleaned up :)

Reply:
Opps thanks for catching that! Ready for another review!